application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -75,6 +75,22 @@
   #set ($keyboardShortcutsEnabled = false)
   #set ($discard = $xwiki.ssx.use('Diagram.DiagramSheet'))
   #set ($discard = $xwiki.jsx.use('Diagram.DiagramEditSheet'))
+  ## Issue #219: Cannot modify diagram title in diagram editor
+  &lt;div class="row xform"&gt;
+    &lt;div class="col-xs-12 col-md-12"&gt;
+      &lt;dl&gt;
+        &lt;dt&gt;
+          &lt;label for="xwikiDocTitleInput"&gt;
```

Comment:
I know you tried to follow the best practice by applying camel-case but I think it's better in this case to use the same id as in standard wiki edit mode. I know that id is awful but I think consistency is more important in this case.